### ISSUE TITLE
Dealing with Query Param Arrays

### ISSUE THREAD

**Issue by jhoover4** [bauerji/flask_pydantic#7]
Another question - how would I go about validating query param arrays? 

For example I have code like this:
```
response = test_client.get("/api/v1/services", query_string={"ids[]": ["5f03539f9472e7f1a153797d"]})

assert response.status_code == 200
```

```
@services_bp.route("", methods=["GET"], strict_slashes=False)
@validate(query=GetObjectIds)
def get_services():
    """
    Retrieves list of all services
    """
    data = request.args.getlist("ids[]")

    if not data:
        services = service_db.all_services()
    else:
        services = service_db.get_services([x for x in data])

    if len(services) > 0:
        return jsonify([service.dict() for service in services])
    else:
        return jsonify(error="No services found with requested ids"), 404
```

I'd like to be able to have a model that says "validate that the `get` query correctly contains an id of lists."

**Comment by jhoover4** [comment]
Happy to help with a PR for this as well if its needed

**Comment by jhoover4** [comment]
Also if you think the format needs to be `ids=1&ids=2&ids=3` let me know

**Comment by bauerji** [comment]
I agree, the format should be like this `ids=1&ids=2&ids=3`. I am sorry I don't think I will have time to implement this feature this week or so. So don't hesitate to create PR on your own if you wish.

**Comment by jhoover4** [comment]
Okay, doing it as `ids=1&ids=2&ids=3` is fine! I'm just wondering if I'm doing something wrong? I keep getting:

```
{'validation_error': {'query_params': [{'loc': ['ids'], 'msg': 'value is not a valid list', 'type': 'type_error.list'}]}}
```

No problem if you don't have time, just want to make sure I'm not missing something before I open a PR

**Comment by bauerji** [comment]
Query params validation is performed [here](https://github.com/bauerji/flask_pydantic/blob/master/flask_pydantic/core.py#L129) 

As you can see, the query arguments `ImutableMultiDict([('ids', '1'), ('ids', '2'), ('ids', '3')])` (in your case) are passed to pydantic's query model ([here](https://github.com/bauerji/flask_pydantic/blob/master/flask_pydantic/core.py#L131)). There is the problem.

What we need in this case is something like this
```
q = query({"ids": request.args.getlist("ids")})
```
but that is not easily possible with current code structure.

**Comment by jhoover4** [comment]
Gotcha. I'll see what I can do

**Comment by bauerji** [comment]
Hi @jhoover4,

I implemented the array query parameters feature. It should work as intended now.

Jirka

**Comment by bauerji** [comment]
This feature is released in version `0.1.0`. Feel free to reopen this issue or create another one if you are not satisfied with implementation.
